galaxy/accounts/models.py

```diff
@@ -67,11 +69,14 @@ class CustomUser(AbstractBaseUser, PermissionsMixin, DirtyMixin):
     date_joined = models.DateTimeField(_('date joined'), default=timezone.now)
 
     karma = models.IntegerField(default=0, db_index=True)
-    github_avatar = models.CharField(_('github avatar'), max_length=254, blank=True)
-    github_user = models.CharField(_('github user'), max_length=254, blank=True)
-    cache_refreshed = models.BooleanField(_('cache refreshed'), default=False)
-
-    objects = UserManager()
+    github_avatar = models.CharField(
```

Comment:
I'm confused. I thought we were changing `github_user` and `github_avatar` to be properties, with the results pulled from the social account table.

Reply:
According to discussion in slack, let's rename it to `avatar` as a part of this PR.